### PR TITLE
[EDU-5525] Update CLI frameworks guides

### PR DESCRIPTION
**Pull request by guiafonso-ol**
### Changes

Updates the steps and notes in framework-specific guides in the CLI docs. It also creates the guides for the frameworks listed in out framework compatibility page and restructures the CLI docs menu to make the navigation easier and more logical.

### CODE REVIEW

File: src/content/docs/en/pages/build-journey/develop-with-azion/frameworks/framework-vite.mdx [hunk 1]
```diff
@@ -37,42 +37,49 @@ Before getting started, you must have:
 
 ---
 
-## Initializing an Vite project on the edge
+## Initializing an VitePress project on the edge
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/develop-with-azion/frameworks/framework-vite.mdx:40]:
```suggestion
## Initializing a VitePress project on the edge
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-docusaurus.mdx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+namespace: docs_framework_docusaurus
+menu_namespace: cliMenuAlpha
+---
+import Tag from 'primevue/tag';
+
+<Tag severity="info" client:only="vue">
+  Aplicação estática
+</Tag>
+
+O Docusaurus é um gerador de sites estáticos, construindo uma `single page application` com navegação rápida no lado do cliente e baseado em React para tornar o site interativo.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-docusaurus.mdx:20]:
```suggestion
O Docusaurus é um gerador de sites estáticos, construindo uma **single page application** com navegação rápida no lado do cliente e baseado em React para tornar o site interativo.
```

File: src/content/docs/en/pages/build-journey/develop-with-azion/frameworks/framework-docusaurus.mdx [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+menu_namespace: cliMenuAlpha
+---
+
+import Tag from 'primevue/tag';
+
+<Tag severity="info" client:only="vue">
+  Static application
+</Tag>
+
+Docusaurus is a static-site generator, building a single-page application with fast client-side navigation and relying on React to make the site interactive.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/develop-with-azion/frameworks/framework-docusaurus.mdx:22]:
```suggestion
Docusaurus is a static-site generator that builds a **single-page application** with fast client-side navigation and relies on React to make the site interactive.
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-gatsby.mdx [hunk 4]
```diff
@@ -0,0 +1,156 @@
+---
+title: Como realizar o deploy de uma aplicação Jamstack com Gatsby
+description: >-
+  Descubra como implantar e gerenciar projetos Gatsby na plataforma Edge da Azion
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-gatsby.mdx:4]:
```suggestion
  Descubra como implantar e gerenciar projetos Gatsby na Plataforma de Edge da Azion
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-hugo.mdx [hunk 5]
```diff
@@ -0,0 +1,156 @@
+---
+title: Como realizar o deploy de uma aplicação Jamstack com Hugo
+description: >-
+  Descubra como implantar e gerenciar projetos Hugo na plataforma Edge da Azion
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-hugo.mdx:4]:
```suggestion
  Descubra como implantar e gerenciar projetos Hugo na Plataforma de Edge da Azion
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-docusaurus.mdx [hunk 6]
```diff
@@ -0,0 +1,156 @@
+---
+title: Como realizar o deploy de uma aplicação Jamstack com Docusaurus
+description: >-
+  Descubra como implantar e gerenciar projetos Docusaurus na plataforma Edge da Azion
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-docusaurus.mdx:4]:
```suggestion
  Descubra como implantar e gerenciar projetos Docusaurus na Plataforma de Edge da Azion
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-eleventy.mdx [hunk 7]
```diff
@@ -0,0 +1,156 @@
+---
+title: Como realizar o deploy de uma aplicação Jamstack com Eleventy
+description: >-
+  Descubra como implantar e gerenciar projetos Eleventy na plataforma Edge da Azion
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-eleventy.mdx:4]:
```suggestion
  Descubra como implantar e gerenciar projetos Eleventy na Plataforma de Edge da Azion
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-hono.mdx [hunk 8]
```diff
@@ -0,0 +1,156 @@
+---
+title: Como realizar o deploy de uma aplicação Jamstack com Hono
+description: >-
+  Descubra como implantar e gerenciar projetos Hono na plataforma Edge da Azion
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-hono.mdx:4]:
```suggestion
  Descubra como implantar e gerenciar projetos Hono na Plataforma de Edge da Azion
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-docusaurus.mdx [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+
+- Uma conta na [plataforma da Azion](/pt-br/documentacao/produtos/contas/criar-uma-conta/) com o produto [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) habilitado.
+- [A versão mais recente do Azion CLI instalada](/pt-br/documentacao/produtos/azion-cli/visao-geral/).
+- Um editor de código.
+- Acesso ao terminal.
+- Node.js ≥ 18 instalado.
+
+---
+
+## Inicializar um projeto Docusaurus
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-docusaurus.mdx:38]:
```suggestion
## Inicialize um projeto Docusaurus
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-hugo.mdx [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+
+- Uma conta na [plataforma da Azion](/pt-br/documentacao/produtos/contas/criar-uma-conta/) com o produto [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) habilitado.
+- [A versão mais recente do Azion CLI instalada](/pt-br/documentacao/produtos/azion-cli/visao-geral/).
+- Um editor de código.
+- Acesso ao terminal.
+- Node.js ≥ 18 instalado.
+
+---
+
+## Inicializar um projeto Hugo
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-hugo.mdx:38]:
```suggestion
## Inicialize um projeto Hugo
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-jekyll.mdx [hunk 11]
```diff
@@ -0,0 +1,159 @@
+---
+title: Como realizar o deploy de uma aplicação Jamstack com Jekyll
+description: >-
+  Descubra como implantar e gerenciar projetos Jekyll na plataforma Edge da Azion
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-jekyll.mdx:4]:
```suggestion
  Descubra como implantar e gerenciar projetos Jekyll na Plataforma de Edge da Azion
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-jekyll.mdx [hunk 12]
```diff
@@ -0,0 +1,159 @@
...
+
+- Uma conta na [plataforma da Azion](/pt-br/documentacao/produtos/contas/criar-uma-conta/) com o produto [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) habilitado.
+- [A versão mais recente do Azion CLI instalada](/pt-br/documentacao/produtos/azion-cli/visao-geral/).
+- Um editor de código.
+- Acesso ao terminal.
+- Node.js ≥ 18 instalado.
+
+---
+
+## Inicializar um projeto Jekyll
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-jekyll.mdx:38]:
```suggestion
## Inicialize um projeto Jekyll
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-svelte.mdx [hunk 13]
```diff
@@ -0,0 +1,163 @@
+---
+title: Como realizar o deploy de uma aplicação Jamstack com Svelte
+description: >-
+  Descubra como implantar e gerenciar projetos Svelte na plataforma Edge da Azion
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-svelte.mdx:4]:
```suggestion
  Descubra como implantar e gerenciar projetos Svelte na Plataforma de Edge da Azion
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-svelte.mdx [hunk 14]
```diff
@@ -0,0 +1,163 @@
...
+
+- Uma conta na [plataforma da Azion](/pt-br/documentacao/produtos/contas/criar-uma-conta/) com o produto [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) habilitado.
+- [A versão mais recente do Azion CLI instalada](/pt-br/documentacao/produtos/azion-cli/visao-geral/).
+- Um editor de código.
+- Acesso ao terminal.
+- Node.js ≥ 18 instalado.
+
+---
+
+## Inicializar um projeto Svelte
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-svelte.mdx:38]:
```suggestion
## Inicialize um projeto Svelte
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-vite.mdx [hunk 15]
```diff
@@ -35,40 +35,45 @@ Antes de começar, você deve ter:
 
 ---
 
-## Inicializar um projeto Vite
+## Inicializar um projeto Press
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-vite.mdx:38]:
```suggestion
## Inicialize um projeto VitePress
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/build-journey/develop-with-azion/frameworks/framework-angular.mdx [hunk 16]
```diff
@@ -44,34 +44,32 @@ Before getting started, you must have:
   azion init
 ```
 
-2. Accept the suggested name for your project, or enter a name:
+2. Give your project a name, or press `enter` to accept the given suggestion:
 
 ```sh 
-(Hit enter to accept the suggested name in parentheses) Your project's name:  (dynamic_caligari)
+? Your application's name:  (black-thor)
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/build-journey/develop-with-azion/frameworks/framework-angular.mdx:50]: "?" Não sei se está correto esse ponto de interrogação.

**Review by gabriel-alves-azion — APPROVED** [review]

**Review by robsongajunior — APPROVED** [review]

**Pull request merged**